core/storage/activity/gae_models_test.py

```diff
@@ -34,6 +34,12 @@ def test_get_deletion_policy(self):
             activity_models.ActivityReferencesModel.get_deletion_policy(),
             base_models.DELETION_POLICY.NOT_APPLICABLE)
 
+    def test_has_reference_to_user_id(self):
+        self.assertFalse(
+            activity_models.ActivityReferencesModel
+            .has_reference_to_user_id('id_x')
+        )
```

Comment:
I think that when there is a multiline statement between `()` it is better to keep it this way for readability (in this case the method ends right after so it would still be quite readable but if there was another statement it would kind of melt together.

Reply:
But thi is probably for a different discussion. I can edit all the occurences of this before merging this.